### ISSUE TITLE
pre-commit configuration for TruffleHog is outdated

### ISSUE THREAD

**Issue by MrSecure** [toniblyx/prowler#5369]
### Steps to Reproduce

Configure environment following the Developer Guide

`git commit` emits the following message:

```console
git commit
[WARNING] hook id `trufflehog` uses deprecated stage names (commit, push) which will be removed in a future version.  run: `pre-commit migrate-config` to automatically fix this.
check for merge conflicts................................................Passed
...
```

### Expected behavior

The pre-commit tasks will execute without generating warnings about their configuration.

### Actual Result with Screenshots or Logs

![Screenshot 2024-10-10 at 13 28 16](https://github.com/user-attachments/assets/6e098857-a2ed-43d4-8de3-32816265ea3c)

### How did you install Prowler?

Cloning the repository from github.com (git clone)

### Environment Resource

Workstation

### OS used

Mac OS

### Prowler version

Prowler 4.5.0 (You are running the latest version, yay!)

### Pip version

pip 24.2

### Context

This impacts only the development workflow, not end-user actions.

**Comment by MrCloudSec** [comment]
Hey @MrSecure, I could not reproduce your error, did you install Prowler's dependencies with `poetry install` when cloning the repo?

**Comment by jfagoagas** [comment]
Hello @MrSecure this was fixed yesterday at #5360, pointed out here https://github.com/prowler-cloud/prowler/pull/5360#discussion_r1795355250.

I'll close the issue. Thanks for using Prowler 🚀
